mixer/pkg/server/server.go

```diff
@@ -120,6 +120,14 @@ func newServer(a *Args, p *patchTable) (*Server, error) {
 	}
 
 	s := &Server{}
```

Comment:
Still need to define new variable `s` here, can not reuse return value `server`, because it will be set to nil after return if there is error.